runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java

```diff
@@ -223,7 +223,9 @@
 		// Taverley
 		STILE,
 		// Asgarnian Ice Dungeon
-		STEPS
+		STEPS,
+		// Wyvern Cave
```

Comment:
Probably good to mention that this is the Fossil Island wyvern cave

Reply:
Fixed